sandbox-backend/src/main/java/com/exadel/sandbox/team5/service/impl/OrderServiceImpl.java

```diff
@@ -111,4 +113,9 @@ public OrderDto createOrder(CreateOrder createOrder) {
     public Map<String, String> getOrdersByCategories() {
         return entityDao.getAllOrdersForCategories().stream().collect(Collectors.toMap(Pair::getFirst, Pair::getSecond));
     }
+
+    public String getCurrentUsername() {
```

Comment:
Please extract this method to something like `SecurityUtils`

Reply:
Done.